apps/openmw/mwmechanics/character.cpp

```diff
@@ -1227,8 +1227,9 @@ bool CharacterController::updateCreatureState()
 
                 mAttackStrength = std::min(1.f, 0.1f + Misc::Rng::rollClosedProbability());
 
+                // Note: for non-bipedal creatures attack strength seems to be 0 during pitch calculation
```

Comment:
It's not 0, it's the same as attack strength (and is 0 if the creature misses and attack strength is 0), but Hrnchamd said swish sound is played on swing release in any instance, which is curious.

Reply:
Unfortunately, we do not know if attack will hit or miss when we start attack animation and start to play the swish sound.